src/components/votingV2/votingV2.css

```diff
@@ -0,0 +1,95 @@
+@import '../app/variables.css';
+
+:root {
+  --submit-button-height: 55px;
+  --filters-height: 55px;
+}
+
+.wrapper {
+  width: 100%;
+  box-shadow: var(--normal-shadow);
+  margin: 0;
+
+  & > aside {
+    padding-left: 0;
+  }
+
+  & > section {
+    padding-right: 0;
+  }
+}
+
+.sideBar {
+  padding-right: 0px;
+}
+
+.votes {
+  padding-left: 0px;
+  background-color: var(--color-white);
+}
+
+@media (--medium-viewport) {
+  .wrapper {
+    margin: 0;
+
+    & > section,
+    & > aside {
+      padding: 0;
+      width: 100%;
+    }
+
+    & > section {
```

Comment:
Is this still being used on the new delegates tab?

Reply:
yes